st2reactor/st2reactor/rules/worker.py

```diff
@@ -26,13 +26,20 @@ def process(self, instance):
         trigger = instance['trigger']
         payload = instance['payload']
 
-        trigger_instance = container_utils.create_trigger_instance(
-            trigger,
-            payload or {},
-            datetime.datetime.utcnow())
-
-        if trigger_instance:
-            self.rules_engine.handle_trigger_instance(trigger_instance)
+        try:
+            trigger_instance = container_utils.create_trigger_instance(
```

Comment:
Kinda annoying that this code returns `None` instead of throwing, it's probably just hiding a programmer error  / invalid usage somewhere.

Anyways.